README.md

```diff
@@ -127,8 +127,11 @@ Example:
 
 ### Pull Requests
 
-- Pull Request title should be the same as commit message
+- Pull Request title should be the same as the commit message
 - Only one commit should be present in one Pull Request
+- Progress not perfection. In the review, we suggest what can be improved. If the issue does not affect the functionality it is up to the owner of the PR if he wants to implement it.
+- All comments need to be answered before the approval.
+- If you approve the PR and you have not found an issue please try to leave a comment.
```

Comment:
So if everything is perfect for me, then besides the approve I need to leave a comment? Could you give an example? 
e.g.  "Everything is ok for me" ?

Reply:
IMO giving approve means "your changes look good" by itself, thus the additional comment will be redundant.

Reply:
@michal-kucznerowicz  That's my point ☝️

Reply:
Fine with me.

Reply:
@MarekMacko @michal-kucznerowicz The requirement has been removed.